extensions/jdbc/jdbc-mariadb/deployment/src/main/java/io/quarkus/jdbc/mariadb/deployment/JDBCMariaDBProcessor.java

```diff
@@ -25,10 +26,9 @@ FeatureBuildItem feature() {
     }
 
     @BuildStep
-    void registerDriver(BuildProducer<JdbcDriverBuildItem> jdbcDriver,
-            SslNativeConfigBuildItem sslNativeConfigBuildItem) {
```

Comment:
`SslNativeConfigBuildItem` was never used, so I removed it. I might be missing something, though.

Reply:
Sometimes we have unused builditems to express ordering restrictions, but this shouldn't be the case. +1 to cleanup